### DISCUSSION TITLE
Detect timestamp of the first word

### DISCUSSION THREAD

**Discussion by LostArrows27** [openai/whisper#1603] [Q&A] [answered]
Hi guys, im using some Javascript library of Whisper in order to make subtitle for video. Is there anyway i can detect time of the first word spoken in the video cause when i test with this [audio file](https://cbvwrikamnjrgcobbmii.supabase.co/storage/v1/object/public/avatar/hyouka.mp3) it only gives me the first conversation from 0:00s to 10:00s meanwhile from 0:00s to 7:00s is just background music.

**Comment by glangford** [comment, marked as answer]
Have you tried `--word_timestamps True`? This is listed as an experimental feature, and it doesn't always give the correct timestamp at the beginning of the file. But it is your best option. See also

* https://github.com/openai/whisper/discussions/435
